packages/react-instantsearch/examples/reactnative/App.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ * @flow
+ */
+
+import React from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  ListView,
+  TextInput,
+  Image,
+} from 'react-native';
+import {InstantSearch} from 'react-instantsearch/native';
+import {connectSearchBox, connectInfiniteHits} from 'react-instantsearch/connectors';
+
+const styles = StyleSheet.create({
+  maincontainer: {
+    paddingTop: 20,
+    paddingBottom: 10,
+  },
+  item: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    backgroundColor: 'white',
+  },
+});
+
+export default React.createClass({
+  render() {
+    return (
+      <View style={styles.maincontainer}>
+        <InstantSearch
+          className="container-fluid"
+          appId="latency"
+          apiKey="6be0576ff61c053d5f9a3225e2a90f76"
+          indexName="ikea"
+        >
+          <SearchBox/>
+          <Hits/>
+        </InstantSearch>
+      </View>
+    );
+  },
+});
+
+const SearchBox = connectSearchBox(React.createClass({
```

Comment:
React.createClass is kinda old school. People tend to favor ES6 class or functions #notBlocking :)